src/web-ui/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This script is called during the build stage in CodePipeline.
+# It generates ./dist output folder of web-ui Vue application.
+# If $LAYER0_DEPLOY_TOKEN is present it generates ./.layer0 output folder for deployment on Layer0.
+
+if [ $LAYER0_DEPLOY_TOKEN = "NONE" ]; then
```

Comment:
is it NONE or empty?

because in src/web-ui/.env.template there is:
LAYER0_DEPLOY_TOKEN=

which seems to be a default to empty?

Reply:
Should be `NONE` by default. I added that.